Gopkg.toml

```diff
@@ -34,7 +34,7 @@ required = [
 
 [[constraint]]
   name = "sigs.k8s.io/cluster-api"
-  revision = "9f86c61769dced58fbc5b148209dccbdb1cf9687"
+  revision = "45f1c93260140936c610e56575d7505ba3d52444"
```

Comment:
@tariq1890 -- I'm onboard with that as well.

Reply:
I would definitely prefer a version here rather than a commit, but seems like cluster-api has some important fixes to incorporate. Also considering the alpha nature of cluster-api, i feel we should keep the commit here (meaning this works with this commit as of today), there are many breaking changes in master (like for example recent providerConfig -> providerSpec changes). I want to keep it to a specific commit, but agree with Tariq's comment of showing our intent than a revision.

Reply:
@tariq1890 / @awesomenix -- thinking through it again, I think it's fine to leave the commit, instead of the branch. When we do version bumps, we can issue that in a separate PR, instead of potentially introducing breaking changes during a `dep ensure`.

I'm going to resolve this convo, but Tariq, feel free to reopen, if you're not okay with that.